insertMoviesToDB.ts

```diff
@@ -0,0 +1,36 @@
+import{ Movie } from "./model/Movie" 
+import fs from 'fs';
+
+export async function insertMoviesData(): Promise<void> {
+    const movies = JSON.parse(fs.readFileSync('./movies.json', 'utf8'));
+  
+    for (const movie of movies) {
```

Comment:
Iterating the movies in a for loop can be an expencive operation, especially if there a lots of movies. Try using Promise.all insted.
```javascript
await Promise.all(movies.map(movie => {
...
return newMovie.save(); 
}))
```